tests/playTests.sh

```diff
@@ -184,10 +184,16 @@ fi
 
 
 println "\n===>  simple tests "
-
+datagen
```

Comment:
Why did you add this call to `datagen`? This will output a ton of garbage to `stdout`.

Reply:
To test whether the function of the Datagen without parameters by default is normal.

Reply:
That's already tested by the following line.